sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/BloomFilterBenchmark.scala

```diff
@@ -81,8 +80,51 @@ object BloomFilterBenchmark extends SqlBasedBenchmark {
     }
   }
 
+  private def writeParquetBenchmark(): Unit = {
+    withTempPath { dir =>
+      val path = dir.getCanonicalPath
+
+      runBenchmark(s"Parquet Write") {
+        val benchmark = new Benchmark(s"Write ${scaleFactor}M rows", N, output = output)
+        benchmark.addCase("Without bloom filter") { _ =>
+          df.write.mode("overwrite").parquet(path + "/withoutBF")
+        }
+        benchmark.addCase("With bloom filter") { _ =>
+          df.write.mode("overwrite")
+            .option(ParquetOutputFormat.BLOOM_FILTER_ENABLED + "#value", true)
```

Comment:
Please debug with this code path. This might be insufficient to enable bloom filters in Parquet library for some reason. The usual suspects are 1) the parameter is not handed over correctly to Parquet, 2) the parameter needs another parameters, 3) this specific data type is not supported yet in Parquet.
```scala
.option(ParquetOutputFormat.BLOOM_FILTER_ENABLED + "#value", true)
```

cc @ggershinsky since he is a Parquet committer.

Reply:
cc @chenjunjiedada